src/H5D.c

```diff
@@ -1277,6 +1441,84 @@ H5Dwrite_async(const char *app_file, const char *app_func, unsigned app_line, hi
     FUNC_LEAVE_API(ret_value)
 } /* end H5Dwrite_async() */
 
+/*-------------------------------------------------------------------------
+ * Function:	H5Dwrite_multi
+ *
+ * Purpose:	Multi-version of H5Dwrite(), which writes selections from
+ *              application memory BUFs into multiple datasets in a file.
+ *
+ * Return:	Non-negative on success/Negative on failure
+ *
+ * Programmer:	Jonathan Kim  Nov, 2013
+ *
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5Dwrite_multi(size_t count, hid_t dset_id[], hid_t mem_type_id[], hid_t mem_space_id[],
+               hid_t file_space_id[], hid_t dxpl_id, const void *buf[])
+{
+    herr_t ret_value = SUCCEED; /* Return value */
+
+    FUNC_ENTER_API(FAIL)
+    H5TRACE7("e", "z*i*i*i*ii**x", count, dset_id, mem_type_id, mem_space_id, file_space_id, dxpl_id, buf);
+
+    if (count <= 0)
```

Comment:
Same here about changing to `if (count == 0)`.

Reply:
Fixed, thanks